test/unit/merge.js

```diff
@@ -973,7 +973,7 @@ describe('Merge', function () {
           moveTo: qux._id,
           path: baz.path,
           remote: baz.remote,
-          sides: {local: 1, remote: 2},
+          sides: {local: 2, remote: 2},
```

Comment:
Interestingly, we were decreasing the local side here...

Reply:
Ah that could explain some weird sides!